recipes-extended/baseboxd-tools/baseboxd-tools/bundle-debug-info

```diff
@@ -249,6 +254,7 @@ COLLECT_OFDPA=0x8
 COLLECT_FRR=0x10
 COLLECT_PORT=0x20
 COLLECT_MSTPD=0x40
+COLLECT_SWITCH=0x80
 
 COLLECT_DEFAULT=$((COLLECT_PKG | COLLECT_LOG | COLLECT_NET | COLLECT_OFDPA | COLLECT_FRR | COLLECT_PORT | COLLECT_MSTPD))
```

Comment:
I would add a comment here to document that one logging function is intentionally left out by default.

Reply:
Added one.